x-pack/filebeat/module/threatintel/_meta/config.yml

```diff
@@ -43,10 +47,10 @@
 
     # How far back to look once the beat starts up for the first time, the value has to be in hours. Each request afterwards will filter on any event newer
     # than the last event that was already ingested.
-    var.first_interval: 24h
+    var.first_interval: 300h
```

Comment:
I'm wondering why the different `first_interval`, some have 300h some 400h. I guess different sources limit how far back can you look?

Reply:
A lot of the sources do not have daily new events, so everyone that tested it the first time was asking why it never returned anything when they started up filebeat.
The values have been set to something better for the specific source, so they are different for all of them.